federation/executor.go

```diff
@@ -408,5 +431,5 @@ func (e *Executor) Execute(ctx context.Context, query *graphql.Query, optionalAr
 	// So we expect only one item in this list
 	res := r[0]
 	deleteKey(res, federationField)
-	return res,optionalResponseMetatdata,  nil
+	return res, optionalResponseMetatdata, nil
```

Comment:
nit: optionalResponseMetatdata --> optionalResponseMetadata

Reply:
oops, let me fix it in another PR.